File: src/main/java/com/github/vootelerotov/intellij/chill/pill/ChillPillAgent.java

package com.github.vootelerotov.intellij.chill.pill;

import com.github.vootelerotov.intellij.chill.pill.transformer.CorruptionMarkerTransformer;
import java.lang.instrument.Instrumentation;

public class ChillPillAgent {

  public static void premain(String args, Instrumentation inst) {
    inst.addTransformer(new CorruptionMarkerTransformer());
  }
}

File: src/main/kotlin/com/github/vootelerotov/intellij/chill/pill/ChillPillAgent.kt
@file:JvmName("ChillPillAgent")

package com.github.vootelerotov.intellij.chill.pill

import com.github.vootelerotov.intellij.chill.pill.transformer.CorruptionMarkerTransformer
import java.lang.instrument.Instrumentation

fun premain(args: String?, inst: Instrumentation) {
  inst.addTransformer(CorruptionMarkerTransformer())
}